opensoc-streaming/OpenSOC-Alerts/src/main/java/com/opensoc/alerts/TelemetryAlertsBolt.java

```diff
@@ -185,10 +184,10 @@ public void execute(Tuple tuple) {
 			JSONArray uuid_list = new JSONArray();
 
 			if (alerts_list == null || alerts_list.isEmpty()) {
-				LOG.trace("[OpenSOC] No alerts detected in: "
+				System.out.println("[OpenSOC] No alerts detected in: "
```

Comment:
Why would we replace `LOG.trace` with `System.out.println`? Isn't that exactly the opposite of what we want?